tailwind.config.ts

```diff
@@ -7,16 +7,16 @@ export const colorPalette = {
   white: '#FFFFFF',
   black: '#000000',
   gray: {
-    100: '#F2F3F7',
-    200: '#E2E2EA',
-    300: '#C6CBD9',
-    400: '#9A9AAF',
-    500: '#7E7E8F',
-    600: '#656575',
-    700: '#535362',
-    800: '#2E2E3A',
-    900: '#262631',
-    950: '#16161E'
+    100: '#F7F7FA',
```

Comment:
we need to ensure when we change these anything using them is updated correctly too